automation/nightly/test-nightly-build.sh

```diff
@@ -10,24 +10,34 @@ wget -q https://dl.google.com/go/go1.22.6.linux-amd64.tar.gz
 tar -C /usr/local -xf go*.tar.gz
 export PATH=/usr/local/go/bin:$PATH
 
-# get latest commits
+# get latest KubeVirt commits
```

Comment:
why `commits` in plural, if you get only the latest one (single)?
same for CDI below